DataConnectors/Forcepoint DLP.json

```diff
@@ -0,0 +1,85 @@
+{
+    "id": "Forcepoint_DLP",
+    "title": "Forcepopint DLP",
+    "publisher": "Forcepoint",
+    "descriptionMarkdown": "The Forcepoint DLP connector allows you to automatically export DLP incident data from Forcepoint DLP into Azure Sentinel in real-time. This enriches visibility into user activities and data loss incidents, enables further correlation with data from Azure workloads and other feeds, and improves monitoring capability with Workbooks inside Azure Sentinel.",
```

Comment:
Add the expanded form of the connector at least once at the beginning as "the Forcepoint DLP (Data Loss Prevention) connector allows ..."

Reply:
Fixed in latest commit